packages/angular/projects/cds-angular/package.json

```diff
@@ -14,6 +14,7 @@
     "url": "https://github.com/vmware/clarity.git"
   },
   "peerDependencies": {
+    "@types/resize-observer-browser": "^0.1.5",
```

Comment:
Core uses the latest TS which has the proper observer types. Possibly clr-angular won't need this when on Angular 12?